zkevm-circuits/src/util/cell_manager_strategy.rs

```diff
@@ -182,26 +183,60 @@ impl CellManagerStrategy for CMFixedWidthStrategy {
         }
         data
     }
+
+    fn query_cell_value(
+        &mut self,
+        _columns: &mut CellManagerColumns,
+        _cell_type: CellType,
+    ) -> super::cell_manager::CellValueOnly {
+        unimplemented!()
+    }
+
+    fn query_cell_with_affinity<F: Field>(
+        &mut self,
+        _columns: &mut CellManagerColumns,
+        _meta: &mut ConstraintSystem<F>,
+        _cell_type: CellType,
+        _affinity: Self::Affinity,
+    ) -> Cell<F> {
+        unimplemented!()
+    }
+
+    fn query_cell_value_with_affinity(
+        &mut self,
+        _columns: &mut CellManagerColumns,
+        _cell_type: CellType,
+        _affinity: Self::Affinity,
+    ) -> super::cell_manager::CellValueOnly {
+        unimplemented!()
+    }
 }
 
-// TODO: This strategy is unfinished.
+#[derive(Debug, Clone)]
 pub(crate) struct CMFixedHeigthStrategy {
```

Comment:
I noticed a typo here. But we can also leave it as a good first issue.
```suggestion
pub(crate) struct CMFixedHeightStrategy {
```

Reply:
Done